Aspire Budgeting/ContentView.swift

```diff
@@ -35,6 +33,7 @@ struct ContentView: View {
     VStack {
       if isLoggedOut {
         SignInView()
+          .frame(maxHeight: .infinity)
           .animation(Animation.spring().speed(1.0)).transition(.move(edge: .trailing))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/080f65df2311b45c8d4b592dff399d3b1bb01b5f/Aspire Budgeting/ContentView.swift#L37"></span>Chained function calls should be either on the same line, or one per line.<br />
<code>multiline_function_chains</code> <code>ContentView.swift:37</code></td>
    </tr>
  </tbody>
</table>